rust/ql/test/extractor-tests/canonical_path/canonical_paths.expected

```diff
@@ -36,13 +36,13 @@ canonicalPaths
 | regular.rs:51:5:51:18 | Use | None | None |
 | regular.rs:57:1:63:1 | fn enum_match | repo::test | crate::regular::enum_match |
 resolvedPaths
-| anonymous.rs:27:17:27:30 | OtherStruct {...} | repo::test | {0}::OtherStruct |
+| anonymous.rs:27:17:27:30 | OtherStruct {...} | None | None |
```

Comment:
This doesn't seem right. I'm not sure why the renaming did this. It doesn't seem to be affecting any of the other tests?